python/lsst/analysis/tools/actions/plot/wholeSkyPlot.py

```diff
@@ -365,13 +391,50 @@ def makePlot(
                     zorder=100,
                 )
 
+        ax.set_aspect("equals")
+        axPos = ax.get_position()
+        ax1 = fig.add_axes([0.73, 0.25, 0.20, 0.47])
+
+        if np.sum(np.isfinite(data["z"])) > 0:
+            ax1.hist(data["z"], bins=len(data["z"] / 10), color=stars_color(), histtype="step")
+        else:
+            ax1.text(0.5, 0.5, "Data all NaN/Inf")
+        ax1.set_xlabel("Metric Values")
+        ax1.set_ylabel("Number")
+        ax1.yaxis.set_label_position("right")
+        ax1.yaxis.tick_right()
+
+        if self.addThreshold and dataName in metricDefs:
+            ax1.axvline(lowThreshold, color=accent_color())
```

Comment:
Again, will we always have upper and lower thresholds?
What if one of these is Nan or None?